pkg/features/system/collector.go

```diff
@@ -369,3 +386,21 @@ func (c *systemCollector) collectSatelites(client *rpc.Client, ch chan<- prometh
 		ch <- prometheus.MustNewConstMetric(hardwareInfoDesc, prometheus.GaugeValue, float64(1), l...)
 	}
 }
+
+func (c *systemCollector) collectLicense(client *rpc.Client, ch chan<- prometheus.Metric, labelValues []string) {
+	r := &licenseInformation{}
+	err := client.RunCommandAndParse("show system license usage", r)
+
+	if err == nil {
+		for i := range r.LicenseInfo.License {
+			licenseLabels := append(labelValues,
+				strings.ToLower(r.LicenseInfo.License[i].Name),
+				strings.ToLower(r.LicenseInfo.License[i].Description),
```

Comment:
I wouldn't add the description to the label stack, I think we don't need that there.
If I want to know what a specific license is or does I can look that up somewhere else, don't think we need that information in the monitoring system.

Reply:
I agree we don't absolutely need to have it there but imho it has potential to make life a little easier. In addition, it won't hurt in terms of cardinality, since name and description are a 1:1 match.

Reply:
As long as the description doesn't change in a future update :D

But yeah, it probably won't hurt us to have it in there.